### PR TITLE
[Synfig Studio/Windows] Fixed issue with rendering to file when using non-Latin filenames

### PR DESCRIPTION
**Pull request by ice0**
Linked issue: https://github.com/synfig/synfig/issues/1885

### CODE REVIEW

**Comment by ice0** [comment]
@rodolforg I've already checked it on Windows, so only code review.

File: synfig-core/src/modules/mod_ffmpeg/trgt_ffmpeg.cpp [hunk 1]
```diff
@@ -195,25 +195,21 @@ ffmpeg_trgt::init(ProgressCallback *cb=NULL)
...
 
 	String ffmpeg_binary_path;
-	std::list< String > binary_choices;
-	binary_choices.push_back("ffmpeg");
-	binary_choices.push_back("avconv");
-	std::list< String >::iterator iter;
-	for(iter=binary_choices.begin();iter!=binary_choices.end();iter++)
-	{
+	std::list<std::string> binary_choices = {"ffmpeg", "avconv"};
+	for(const auto& iter : binary_choices) {
```
Comment (rodolforg) [inline synfig-core/src/modules/mod_ffmpeg/trgt_ffmpeg.cpp:199]: It isn't an iterator but the element "itself", so I think `iter` isn't a good variable name.

File: synfig-core/src/modules/mod_ffmpeg/trgt_ffmpeg.cpp [hunk 2]
```diff
@@ -195,25 +195,21 @@ ffmpeg_trgt::init(ProgressCallback *cb=NULL)
...
-		if (binary_path != "")
+		if (!binary_path.empty()) {
 			binary_path = etl::dirname(binary_path)+ETL_DIRECTORY_SEPARATOR;
-		binary_path += *iter+".exe";
-		if( access( binary_path.c_str(), F_OK ) != -1 ) {
-			binary_path = "\"" + binary_path + "\"";
-			ffmpeg_binary_path = binary_path;
+		}
+		binary_path.append(iter).append(".exe");
+		if (g_access(binary_path.c_str(), F_OK ) != -1 ) {
```
Comment (rodolforg) [inline synfig-core/src/modules/mod_ffmpeg/trgt_ffmpeg.cpp:207]: I imagine this is the issue solver? `g_access` instead of `access` ?

**Review by rodolforg — CHANGES_REQUESTED** [review]

File: synfig-core/src/modules/mod_ffmpeg/trgt_ffmpeg.cpp (hunk 2 above)
Comment (ice0) [reply synfig-core/src/modules/mod_ffmpeg/trgt_ffmpeg.cpp:207]: Yes.

File: synfig-core/src/modules/mod_ffmpeg/trgt_ffmpeg.cpp (hunk 1 above)
Comment (ice0) [reply synfig-core/src/modules/mod_ffmpeg/trgt_ffmpeg.cpp:199]: Ok

**Comment by ice0** [comment]
I decided to split this PR to avoid mixing code refactoring with fixing the problem.

**Review by rodolforg — DISMISSED** [review]

**Pull request merged**